core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -468,7 +468,8 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
       filters.forall(TiUtil.isSupportedFilter(_, source, blacklist)) &&
       groupingExpressions.forall(TiUtil.isSupportedGroupingExpr(_, source, blacklist)) &&
       aggregateExpressions.forall(TiUtil.isSupportedAggregate(_, source, blacklist)) &&
-      !aggregateExpressions.exists(_.isDistinct)
+      !aggregateExpressions.exists(_.isDistinct) &&
```

Comment:
are planning a better fix? If so, we may add a todo here.

Reply:
fixed

Reply:
> are planning a better fix? If so, we may add a todo here.

fixed